app/themes/covid/templates/thank-you.html

```diff
@@ -37,7 +37,7 @@ <h1 class="u-fs-l" data-qa="submission-successful-title">{{ _("Submission succes
     {% if view_submission_url %}
     <p class="u-mb-m@s u-fs-r" data-qa="view-submission"><a href={{view_submission_url}} data-ga="click"
             data-ga-category="view-submission Covid" data-ga-action="Goto covid view-submission page"
-            data-ga-label="Goto covid survey view-submission page">{{ _("View and print a copy of your answers") }}</a>
+            data-ga-label="Goto covid survey view-submission page">{{ _("VView, save or print a copy of your answers") }}</a>
```

Comment:
```suggestion
            data-ga-label="Goto covid survey view-submission page">{{ _("View, save or print a copy of your answers") }}</a>
```
Spelling mistake here

Reply:
Jinx!!

Reply:
Damn

Reply:
Got me